### ISSUE TITLE
Ensures ARIA attributes are allowed for an element's role

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#2803]
impact: 
Element location: span[aria-label="Current\ Page\,\ Page\ 1"]
Element source: <span class="page-link" aria-label="Current Page, Page 1" aria-current="true">1</span>

Fix the following: ARIA attribute: aria-label is not well supported. Use a different role attribute or element.

ACCEPTANCE CRITERIA:

- [ ] ARIA attribute: aria-label is not well supported. Use a different role attribute or element.

**Comment by sandbergja** [comment]
Is this ticket still valid?  The pagination on the search results page seems to be following this pattern pretty well: https://www.a11ymatters.com/pattern/pagination/.

Although I did see [this note from Mozilla](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label#associated_roles) that "The aria-label attribute is intended for interactive elements only".

We are using stock blacklight pagination here, so if we want to make a change, we could consider contributing them upstream.

**Comment by kevinreiss** [comment]
Per Jane's comment and earlier review we are deciding to close this one.
